pkg/models/repository_configuration.go

```diff
@@ -78,6 +82,9 @@ func (rc *RepositoryConfiguration) BeforeUpdate(tx *gorm.DB) error {
 	if err := rc.validate(); err != nil {
 		return err
 	}
+	if err := rc.SetLabel(); err != nil {
+		return err
+	}
```

Comment:
I'm not sure that we want to update the label today (I'm not 100% sure we even can in candlepin).  We may want to file an issue to handle that later on and properly update it in candlepin if possible.